scripts/ci/install-go-deps.sh

```diff
@@ -16,15 +16,6 @@ go get -f -u github.com/tebeka/go2xunit
 export PATH=$PATH:$GOPATH/bin
 
 # speedup govendor sync command
-curl -o /tmp/vendor.tgz http://46.231.132.68:8080/v1/AUTH_0ec9e4f4f3044236b4d18536ccfcb182/skydive/vendor/vendor.tar.gz
-
-pushd ${GOPATH}/src/github.com/skydive-project/skydive
-go get -f -u github.com/kardianos/govendor
-govendor sync -n | perl -pe 's|fetch \"(.*)\"$|vendor/\1|g' | sort -u > vendor.fetch.list
-cat vendor.fetch.list | xargs tar -xvzf /tmp/vendor.tgz --exclude "vendor/vendor.json"
-# remove installed
-find vendor/ -mindepth 2 -type f | xargs dirname | sort -u > vendor.installed.list
-echo "package to be removed/cleanup"
-diff -u vendor.fetch.list vendor.installed.list | grep '^\+v' | perl -pe 's|^\+(.*)|\1|' | tee /dev/stdout | xargs -n 1 rm -rf
-rm -f vendor.fetch.list vendor.installed.list
-popd
+mkdir $HOME/.govendor $GOPATH/.cache
+rm -rf $GOPATH/.cache/govendor
```

Comment:
The workspace is cleaned for every build so this line is useless

Reply:
I known it's useless on our CI, but will works all the time, in case using locally